packages/e2e/test/wallet/SingleAddressWallet/txChainHistory.test.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable max-len */
+import { assertTxIsValid } from '../../../../wallet/test/util';
+import { buildTx } from '@cardano-sdk/wallet';
+import { deepEquals, isNotNil } from '@cardano-sdk/util';
+import { env } from '../environment';
+import { filter, firstValueFrom, map, take } from 'rxjs';
+import { getWallet } from '../../../src';
+import { logger } from '@cardano-sdk/util-dev';
+
+export type TestWallet = Awaited<ReturnType<typeof getWallet>>;
+
+describe('tx chain history', () => {
+  let wallet: TestWallet;
+
+  beforeEach(async () => {
+    wallet = await getWallet({ env, logger, name: 'Sending Wallet', polling: { interval: 50 } });
```

Comment:
Other props don't seem to be used
```suggestion
    ({ wallet }) = await getWallet({ env, logger, name: 'Sending Wallet', polling: { interval: 50 } });
```